_data/sites.json

```
@@ -19241,7 +19256,7 @@
         "email": "ciq@digital.go.jp",
         "email_body": "Please delete my Visit Japan Web account.",
         "domains": [
-            "vjw.digital.go.jp"
+            "www.vjw.digital.go.jp"
```

```suggestion
            "vjw.digital.go.jp"
```

This change should not be needed